### DISCUSSION TITLE
Disable color darkening for some vibrant colors

### DISCUSSION THREAD

**Discussion by KodzghlyCZ** [darkreader/darkreader#8053] [Q&A] [answered]
If I enable Dark Reader, it dims all the elements including some vibrant colors, wich makes the page quite ugly. It is usualy some panels or other colorful elements. 

A good example would be the github submit button. If I have darkmode enabled, the button stays vibrant, but Dark Reader causes such elements to dim.

I have tried changing the settings, but nothing is taking any effect. Is there any option for keeping some elements unchanged? Thanks

**Comment by Gusted** [comment, marked as answer]
> If I enable Dark Reader, it dims all the elements including some vibrant colors, wich makes the page quite ugly. It is usualy some panels or other colorful elements.
> 
> A good example would be the github submit button. If I have darkmode enabled, the button stays vibrant, but Dark Reader causes such elements to dim.

I'm not sure what vibrant colors is per your definition, but the mentioned github could because you have github's native dark mode enabled.

Also, we don't provide a option to leave certain elements unchanged as that's a technical nightmare and it will cause more problems than good, as the color scheme of that button will be out of touch with the other elements.
